@ecomplus/storefront-app/src/components/js/EcOrderInfo.js

```diff
@@ -218,6 +230,25 @@ export default {
       }
     },
 
+    buyAgain () {
+      const { localOrder } = this
+      if (localOrder.items) {
+        const { items } = localOrder
+        items.forEach((item, i) => {
+          ecomCart.addItem(item)
+          if (i + 1 === localOrder.items.length && ecomCart.data._id) {
+            ecomCart.save()
```

Comment:
```suggestion
          if (i + 1 === items.length) {
            ecomCart.save()
```

Não precisa verificar `data._id` aqui, o `ecomCart.save()` só atualiza o local storage.
Acho que você pode começar debuggando o `item` aqui.